ksqldb-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedDeserializer.java

```diff
@@ -126,6 +128,10 @@ private static Parser decimalParser(final SqlType sqlType) {
     return v -> DecimalUtil.ensureFit(new BigDecimal(v), decimalType);
   }
 
+  private static Parser dateParser(final SqlType sqlType) {
```

Comment:
Don't you mean this?

```suggestion
  private static Parser timestampParser(final SqlType sqlType) {
```